cmd/medius/common/registry.go

```diff
@@ -16,77 +16,71 @@ import (
 )
 
 type Entry struct {
-	Artifact           api.Artifact
+	Artifacts          []api.Artifact
 	UseForDocs         bool
 	UseForLatest       bool
 	SkipWhenNotFocused bool
 }
 
 var staticRegistry = []Entry{
 	{
-		Artifact:   centos.New("8.4", nil),
+		Artifacts: []api.Artifact{
+			centos.New("8.4", nil),
+		},
 		UseForDocs: false,
 	},
 	{
-		Artifact: centos.New(
-			"7-2009",
-			defaultLabels("u1.small", "centos.7"),
-		),
+		Artifacts: []api.Artifact{
+			centos.New("7-2009", defaultLabels("u1.small", "centos.7")),
+		},
 		UseForDocs: true,
 	},
 	{
-		Artifact: centosstream.New(
-			"9",
-			&docs.UserData{
-				Username: "cloud-user",
-			},
-			defaultLabels("u1.small", "centos.stream9"),
-		),
+		Artifacts: []api.Artifact{
+			centosstream.New("9", &docs.UserData{Username: "cloud-user"}, defaultLabels("u1.small", "centos.stream9")),
+		},
 		UseForDocs: true,
 	},
 	{
-		Artifact: centosstream.New(
-			"8",
-			&docs.UserData{
-				Username: "centos",
-			},
-			defaultLabels("u1.small", "centos.stream8"),
-		),
+		Artifacts: []api.Artifact{
+			centosstream.New("8", &docs.UserData{Username: "centos"}, defaultLabels("u1.small", "centos.stream8")),
+		},
 		UseForDocs: false,
 	},
 	{
-		Artifact: ubuntu.New(
-			"22.04",
-			defaultLabels("u1.small", "ubuntu"),
-		),
+		Artifacts: []api.Artifact{
+			ubuntu.New("22.04", defaultLabels("u1.small", "ubuntu")),
+		},
 		UseForDocs: true,
 	},
 	{
-		Artifact: ubuntu.New(
-			"20.04",
-			defaultLabels("u1.small", "ubuntu"),
-		),
+		Artifacts: []api.Artifact{
+			ubuntu.New("20.04", defaultLabels("u1.small", "ubuntu")),
+		},
 		UseForDocs: false,
 	},
 	{
-		Artifact: ubuntu.New(
-			"18.04",
-			defaultLabels("u1.small", "ubuntu"),
-		),
+		Artifacts: []api.Artifact{
+			ubuntu.New("18.04", defaultLabels("u1.small", "ubuntu")),
+		},
 		UseForDocs: false,
 	},
 	// for testing only
 	{
-		Artifact: generic.New(
-			&api.ArtifactDetails{
-				SHA256Sum:   "cc704ab14342c1c8a8d91b66a7fc611d921c8b8f1aaf4695f9d6463d913fa8d1",
-				DownloadURL: "https://download.cirros-cloud.net/0.6.1/cirros-0.6.1-x86_64-disk.img",
-			},
-			&api.Metadata{
-				Name:    "cirros",
-				Version: "6.1",
-			},
-		),
+		Artifacts: []api.Artifact{
```

Comment:
@0xFelix Do you think we should add here in this commit, another artifact of `cirros` that is arm64? I plan to add arm64 to all artifacts in another commit later.

Reply:
Not required I think, but you can if you want to.

Reply:
I'll handle it in another PR, where we'll add arm64 images.